releasenotes/notes/procfs_override-2fdae813d176cf04.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Allow overriding procfs path. Should allow to collect relevant host metrics
+    in containerized environments. 
```

Comment:
I think we could mention that this affects python checks, and that they should now behave in the same way as Agent 5 wrt the `procfs_path` option